### ISSUE TITLE
Possibility of adding main colors option to rezero theme? 

### ISSUE THREAD

**Issue by ctran046** [CurimuChizu/CC-Themes#11]
Hi, I was wondering if it would be possible to simplify user customization options for the Rezero theme? The only thing I can really figure out is how to change the background image.

Is there a possibility of maybe adding a something like: 

main color: 
sub colors: 

or other user options to be able to customize things more easily? For instance, changing main color to a certain color code would change the main layout elements to that color instead of purple. Sub color would change the minor elements to that color instead of blue. 

Your theme seems to be one of the few ones that actually looks nice and still works but I would love to be able to customize to a different color to fit my background image. I tried replacing the 3 colors that you listed in the css (everything, hovering, and texts) but no matter what I do, there always seems to be purple left over.

**Comment by CurimuChizu** [comment]
@ctran046 Yeah, it is possible but it will take a while. I was planning on doing something like that in the past but I stopped updating this theme a while back after the settings update but if you want to try go ahead, it's very possible. :)
